example/admin.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:firedart/firedart.dart';
+
+/// Demonstrates the Firestore Admin API using a privileged service account.
+/// Operations are made "server side" and bypass all Firestore database rules
+/// that apply to regular users.
+/// YOUR SERVER CODE IS RESPONSIBLE FOR AUTHENTICATING AND AUTHORIZING REQUESTS
+///
+/// This is a subset of the Admin SDK: https://firebase.google.com/docs/admin/setup
+/// It is used for server side batch jobs, or for performing
+/// complex updates that are difficult to implement on the client.
+///
+/// Usage:
+/// Create a service account with permission to access Firestore.
+/// See the console:
+/// https://console.firebase.google.com/u/0/project/_/settings/serviceaccounts/adminsdk
+/// Download and save the service account json file.
+/// ****** DO NOT CHECK THIS FILE INTO git! *****
+/// It is highly privileged and should be kept secret.
+/// Set the environment variable GOOGLE_APPLICATION_CREDENTIALS to the
+/// path to this file.  For example:
+/// GOOGLE_APPLICATION_CREDENTIALS=./service-account.json
+///
+
+const apiKey = 'Project Settings -> General -> Web API Key';
+const projectId = 'Project Settings -> General -> Project ID';
+
+Future main() async {
+  var projectId = 'fluttergolf'; // replace with your project id here
+
+  FirebaseAuth.initialize(apiKey, VolatileStore(),
+      serviceAccount: serviceAccountManager().fromJson(r'''
+{
+  <Project Settings -> Service Accounts -> generate new private key>
+  You can either use serviceAccountManager().fromEnvironmentVariable() to locate the .json you just downloaded
+  for you; provided you have the environment variable set properly GOOGLE_APPLICATION_CREDENTIALS
+  Or you can paste your json here as is.
+  YOU MUST NOT CHECK THIS FILE INTO GIT IF YOU DO THIS.
+}
```

Comment:
This seems like a lot of text that's just repeating what's already in the documentation.

Reply:
Fixed.